lib/data/dataset.js

```diff
@@ -44,12 +44,14 @@ const validateDatasetName = (name) => {
 const getDataset = (xml) => {
   const metaNode = findOne(root('html'), node('head'), node('model'), node('instance'), node(), node('meta'));
   return traverseXml(xml, [
+    findOne(root('html'), node('head'), node('model'))(attr('entities-version')),
     metaNode(findOne(root(), node('entity'))(tree())),
     metaNode(findOne(root(), and(node('entity'), hasAttr('dataset')))(attr('dataset')))
-  ]).then(([ entityTag, datasetName ]) => {
-    // check if <entity> tag exists at all
+  ]).then(([ version, entityTag, datasetName ]) => {
     if (entityTag.isEmpty() && datasetName.isEmpty())
       return Option.none();
+    if (version.isEmpty() || !version.get().startsWith('2022.1.'))
+      throw Problem.user.invalidEntityForm({ reason: 'Version is not supported.' });
```

Comment:
This message will be visible to API users, right? Maybe "Entities specification version not supported?" Could you include the version code from the form?